connector/src/main/java/org/geysermc/connector/GeyserConnector.java

```diff
@@ -136,8 +141,28 @@ private GeyserConnector(PlatformType platformType, GeyserBootstrap bootstrap) {
         if (platformType != PlatformType.STANDALONE) {
             DockerCheck.check(bootstrap);
         }
+        String remoteAddress = config.getRemote().getAddress();
+        int remotePort = config.getRemote().getPort();
+        if((config.isLegacyPingPassthrough() || platformType == PlatformType.STANDALONE) && !remoteAddress.matches(IP_REGEX) && !remoteAddress.equalsIgnoreCase("localhost")){
+            try {
+                InitialDirContext ctx = new InitialDirContext();
+                Attribute attr = ctx.getAttributes("dns:///_minecraft._tcp." + remoteAddress, new String[]{"SRV"}).get("SRV");
+                if (attr.size() > 0) {
+                    String[] record = ((String) attr.get(0)).split(" ");
+                    remoteAddress = record[3];
+                    if(remoteAddress.endsWith(".")){
+                        remoteAddress = remoteAddress.substring(0, remoteAddress.length() - 1);
+                    }
+                    config.getRemote().setAddress(remoteAddress);
+                    config.getRemote().setPort(remotePort = Integer.parseInt(record[2]));
+                    logger.debug(LanguageUtils.getLocaleStringLog("geyser.core.start.srv_record", remoteAddress, String.valueOf(remotePort)));
+                }
+            }catch (NamingException ex){
```

Comment:
Use spacing (`} catch (NamingException ex) {`)to match with the rest of the Geyser codebase.